superduperdb/components/listener.py

```diff
@@ -42,12 +42,12 @@ class Listener(Component):
     type_id: t.ClassVar[str] = 'listener'
 
     def __post_init__(self):
-        super().__post_init__()
         if self.identifier is None and self.model is not None:
```

Comment:
Why do we need to switch this order?

Reply:
I added a check for identifier in the `post_init` method of `Component`. Then if any subclass wants to handle a none identifier,  they need to fix the identifier and then call the super().__post_init()